Sources/NIOFoundationCompat/ByteBuffer-foundation.swift

```diff
@@ -265,6 +265,101 @@ extension ByteBuffer {
         }
         return written
     }
+
+    // MARK: - UUID
+
+    /// Get a `UUID` from the 16 bytes starting at `index`. This will not change the reader index.
+    /// If there are less than 16 bytes starting at `index` then `nil` will be returned.
+    ///
+    /// - Parameters:
+    ///   - index: The starting index of the bytes of interest into the `ByteBuffer`.
+    /// - Returns: A `UUID` value containing the bytes of interest or `nil` if the selected bytes
+    ///            are not readable or there were not enough bytes.
+    public func getUUID(at index: Int) -> UUID? {
+        guard let chunk1 = self.getInteger(at: index, as: UInt64.self),
+              let chunk2 = self.getInteger(at: index + 8, as: UInt64.self) else {
```

Comment:
I think we should use the new multiread here... :)

Reply:
There's no `getMultipleIntegers` unfortunately. I considered reading a slice and then reading integers from that but then we're doing even more checking.

Reply:
want to add it?

Reply:
Yeah, we should add get and set. I just don't have the time right now 😔 (I only opened this PR because I already had the changes locally somewhere else and thought it would be generally useful.)